packages/contracts/src/execute/depositCollateral.ts

```diff
@@ -25,8 +29,12 @@ export async function depositCollateral({
   if (referralCode) {
     return endpoint[
       'depositCollateralWithReferral(bytes12,uint32,uint128,string)'
-    ](bytesSubaccountName, productId, amount, referralCode);
+    ](bytesSubaccountName, productId, toIntegerString(amount), referralCode);
```

Comment:
Why is this using `toIntegerString`, but L37 uses `toBigInt`?

Reply:
Both should use toBigInt actually.